com.civicrm.hrjobroles/js/karma.conf.js

```diff
@@ -67,6 +69,7 @@ module.exports = function (config) {
         ]
       }
     },
+    reporters: argv.reporters ? argv.reporters.split(',') : ['spec'],
```

Comment:
Why dont we keep
`reporters: ['spec'],` 
here in the `karma.conf.js` and in ` uk.co.compucorp.civicrm.hrcore/gulp/test.js`
```js
var reporters = argv.reporters ? argv.reporters.split(',') : configFile.reporters;

  new karma.Server({
    configFile: configFile,
    reporters: reporters,
    singleRun: true
  }, function () {
    cb && cb();
  }).start();
```
This way the repetitive logic can be minimized or almost eliminated and it would still be standard because the "normal" reporter is defined in karma.conf.js but only part related to overriding via jenkins is moved to the main gulpfile.

Reply:
Please see the comment

> There is a lot of repetition due to the fact that every single extension has its separate karma.conf.js file. This should be addressed in a future PR where we will have instead one single conf file that gets adapted dynamically by whichever extension is currently being tested

I don't expect that `reporters` line to be change any time soon, so the repetition won't be a hindrance in the meantime, until we have a unified karma.conf file